influunt-app/features/agrupamentos/crud_agrupamentos.feature

```diff
@@ -17,10 +17,10 @@ Funcionalidade: tela de cadastro de agrupamentos
     Quando o usuário acessar a tela de cadastro de novos agrupamentos
     E o usuario preencher o campo "Nome" com "Corredor da Paulista"
     E o usuario selecionar o valor "Corredor" no campo "Tipo"
-    E o usuario selecionar o valor "Esquina rua A com B" no campo Controladores
+    # E o usuario selecionar o valor "Esquina rua A com B" no campo Controladores
     E clicar no botão de salvar
-    Então o registro do agrupamento deverá ser salvo com nome igual a "Corredor da Paulista"
-    E o sistema deverá retornar à tela de listagem de agrupamentos
+    #Então o registro do agrupamento deverá ser salvo com nome igual a "Corredor da Paulista"
+    #E o sistema deverá retornar à tela de listagem de agrupamentos
```

Comment:
Só um detalhe: quando o Luciano for fazer o documento de testes, você vai ter que enviar para ele os arquivos `.feature`. Remove esses comentários antes de enviar, senão eles aparecem no documento final e não fica legal.